src/main/java/org/traccar/protocol/GatorProtocolEncoder.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2017 - 2019 Anton Tananaev (anton@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.protocol;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.traccar.BaseProtocolEncoder;
+import org.traccar.Main;
+import org.traccar.Protocol;
+import org.traccar.helper.Checksum;
+import org.traccar.model.Command;
+
+public class GatorProtocolEncoder extends BaseProtocolEncoder {
+
+    public GatorProtocolEncoder(Protocol protocol) {
+        super(protocol);
+    }
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
+
+    public static final int GET_POSITION = 0x30;
+    public static final int SET_ENGINE_START = 0x38;
+    public static final int SET_ENGINE_STOP = 0x39;
+
+
+    private ByteBuf engineExecute(String pseudoAddress, int engineState) {
+        ByteBuf buf = Unpooled.buffer(256);
+
+        // Add Header - 24 24
```

Comment:
Please remove most of these comments. Only include comment when absolutely necessary. Don't include comments for something that's immediately obvious from the code.